README.md

```diff
@@ -1,4 +1,4 @@
-# Ramme <img src="static/icon.png" width="70">
+# Ramme(Stephen.Kingsley process) <img src="static/icon.png" width="70">
```

Comment:
this is my fault! I am interesting in this project, so I want to fork a version to create some awesome! I modify it!